css/css-color/predefined-005.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: predefined colorspaces, display-p3, decimal values</title>
+<link rel="author" title="Chris Lilley" href="mailto:chris@w3.org">
+<link rel="help" href="https://drafts.csswg.org/css-color-4/#predefined">
+<link rel="match" href="greensquare-ref.html">
+<meta name="assert" content="Color function with explicit display-p3 value as decimal matches sRGB #008000">
+<style>
+    .test { background-color: red; width: 12em; height: 6em; margin-top:0}
+    .ref { background-color: #008000; width: 12em; height: 6em; margin-bottom: 0}
+    .test {background-color: color(display-p3 0.21508071610847918 0.4922429411940395 0.13085464082643297)}

```

Comment:
My guess is that this test will fail due to minor color matching differences between OSes. I think  there needs to be a way to do slightly fuzzy matching.

Reply:
I didn't think that the precision would affect the test passing: its just a test for sRGB green and currently, many browsers will round to 8 bits/component anyway. The test does not inspect the computed value. Its a simple "see red, no green" test. I simply didn't want to round the values (for example, the sRGB specification had to get fixed because the consequences of rounding to 3 figures were not realized).

Reply:
That said, I'm aware of the need not to compare two floats. But this test is visual, so the fuzziness comes from 

1. the human visual system (if there is less than 1 or 2 ΔE between the sRGB green and the test patch, people won't see a difference
2. the precision with which the underlying used value is presented to the display surface, which might be 10 bits _(and possibly 12bits on a commercial, Rec.2020 studio monitor)_, but certainly not higher than that.

Reply:
Now using 5 significant digits. But as I said before, this is a visual reftest.